packages/push-platform-notifications-controller/tsconfig.build.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist/types",
+    "rootDir": "./src",
+  },
+  "references": [
+    {
+      "path": "../base-controller/tsconfig.build.json"
+    },
```

Comment:
Based on `package.json`, should we add `notifications-controller` and `profile-sync-controller`?

``` suggestion
    {
      "path": "../notifications-controller/tsconfig.build.json"
    },
    {
      "path": "../profile-sync-controller/tsconfig.build.json"
    },
```